introduction.md

```diff
@@ -1,3 +1,9 @@
 A true shinobi never introduces himself. His weapons are his introduction as well as his farewell.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?